### PR TITLE
Add bionic to the apt::key regexp to support Ubuntu 18.04

### LINKED ISSUE #82: Missing repository key for Ubuntu 18.04
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet:5.4.0-2ubuntu3
- Ruby:1:2.5.1
- Distribution:Ubuntu 18.04.1
- Module version:2.2.1

## How to reproduce (e.g Puppet code you use)

class { '::virtualbox':
manage_kernel => false,
manage_package => false,
}

## What are you seeing

W: GPG error: http://download.virtualbox.org/virtualbox/debian bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY A2F683C52980AECF
E: The repository 'http://download.virtualbox.org/virtualbox/debian bionic InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

## What behaviour did you expect instead

No warning message regarding VirtualBox repository

### PR DESCRIPTION
**Pull request by meskaya**
#### Pull Request (PR) description

Add bionic in a regexp used with apt::key to install the right repository key for Ubuntu 18.04

#### This Pull Request (PR) fixes the following issues
    Fixes #82

### CODE REVIEW

**Comment by meskaya** [comment]
Yes it should work if we have both the right repo key and the node architecture in the repo source list

File: metadata.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
         "16.04"
```
Comment (Dan33l) [inline metadata.json]: missing comma
```suggestion
        "16.04",
```

**Comment by alexjfisher** [comment]
@meskaya If you need any help with rebasing and squashing, we’d be happy to see you on IRC (freenode #voxpupuli), or Slack.

**Comment by Dan33l** [comment]
@meskaya when you are thinking rebase and squash are done locally (before to push) , you can use `git log --oneline`. This permit to check the history. And so just after the merge of your previous PR, you should find only one line about this PR (not several).

**Comment by Dan33l** [comment]
Yes ! You did squash of several commits in one commit.
From here : 

```
                  M 
Master branch +---+
                   \
 Feature branch     +---+---+
                        C1  C2
```
We are now here 
```
                   M
 Master branch +---+
                    \
 Feature branch      +---+
                         C=C1+C2
```

But since you cloned Voxpupuli repository our repo moved because we merged your previous PR.
So now, you have to rebase and so your clone will include your previous PR like this : 
```
                       Here merge of previous PR
                   M   v
 Master branch +---+---+
                        \
 Feature branch          +---+
                             C=C1+C2
```
So from this history (`git log --oneline`) :  
```
9411685 Add bionic to the apt::key regexp to support Ubuntu 18.04
2edd695 [blacksmith] Bump version to 2.2.2-rc0
c612d6a Merge pull request #79 from bastelfreak/rel221
c65499a release 2.2.1
059dff6 Merge pull request #78 from voxpupuli/modulesync
```
You should get instead : 
```
9411685 Add bionic to the apt::key regexp to support Ubuntu 18.04   <--- Here this PR
146f396 Merge pull request #81 from meskaya/architecture            <--- Here merge of previous PR
fcd736e Add node architecture to the apt repository
2edd695 [blacksmith] Bump version to 2.2.2-rc0
c612d6a Merge pull request #79 from bastelfreak/rel221
c65499a release 2.2.1
059dff6 Merge pull request #78 from voxpupuli/modulesync
```

**Comment by meskaya** [comment]
Is the code I pushed ok ?

**Pull request merged**